### PR TITLE
fix: Improve Failure Exporter configuration documentation

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Signed-off-by: Mateus Oliveira <msouzaol@redhat.com>

## Describe the behavior changes introduced in this PR

Improve the documentation of Failure exporter configuration.

## Linked Issues

No.

## Testing Instructions

Check the documentation generated by the read the docs job.

### CODE REVIEW

File: docs/GettingStarted/configuration/PelorusExporters.md [hunk 1]
```diff
@@ -204,43 +204,22 @@ When it starts up, you should see information about custom certificate usage, de
 
 ### Labels
```
Comment (mateusoliveira43) [inline docs/GettingStarted/configuration/PelorusExporters.md:205]: @mpryc do you agree with these changes in this section?

File: docs/GettingStarted/configuration/PelorusExporters.md [hunk 2]
```diff
@@ -417,86 +399,3 @@ $ oc label image "sha256:${IMAGE_SHA}" "app.kubernetes.io/name=${NAME}"
 $ oc annotate image "sha256:${IMAGE_SHA}" --overwrite \
      io.openshift.build.commit.date="${EPOCH_TIMESTAMP}"
 ```
-
```
Comment (mateusoliveira43) [inline docs/GettingStarted/configuration/PelorusExporters.md:420]: @mpryc This part I will move to failure exporter file. Do you agree?

**Comment by mateusoliveira43** [comment]
I noticed that we are calling the failure exporter both **Failure Time Exporter** and **Failure Exporter**. I want to standardize to one of them. Which you prefer?

File: docs/GettingStarted/configuration/ExporterFailure.md [hunk 3]
```diff
@@ -97,7 +169,7 @@ An example of not setting the `APP_FIELD` is here:
 05-25-2022 13:16:14 INFO     Collected failure_creation_timestamp{ app=todolist-mariadb-go, issue_number=3 } 1652394664.0
 ```
 
-# ServiceNow failure exporter details
+## Configuring ServiceNow
```
Comment (mateusoliveira43) [inline docs/GettingStarted/configuration/ExporterFailure.md:172]: Still need to work in this section

**Comment by mpryc** [comment]
/lgtm

One note to consider later:
 - move examples under main "Examples" and just reference?
 - in the examples where secrets/jira is used provide some short example how to create one, so it's kind of copy-paste

**Pull request merged**